openapi/paths/usages@{id}.yaml

```
@@ -0,0 +1,89 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+get:
+  x-products:
+    - Core
+  tags:
+    - Usage
+  summary: Retrieve a usage record
+  operationId: GetUsage
+  x-sdk-operation-name: get
+  security:
+    - SecretApiKey: []
+    - JWT: []
+  description: |
+    Retrieves a usage record.
+  responses:
+    '200':
+      description: Usage record retrieved successfully.
```

nitpick: 'successfully' does not add to the sentence 
```suggestion
      description: Usage record retrieved.
```